src/testplane/testplane.ts

```diff
@@ -119,16 +119,15 @@ export class Testplane {
     public async getPostMortemData(): Promise<PostMortemData> {
         const tesptlaneConfig = await this.config();
         const testplaneFailedTestsJsonPath = tesptlaneConfig.getLastFailedTestsJsonPath();
+        const testplaneFailedTestsJsonRootRelativePath = getRootRelativePath(testplaneFailedTestsJsonPath);
```

Comment:
User might specify "cwd", in which we execute testplane.
After testplane run is finished, "testplane failed tests json" file is located inside specified "cwd"
"getRootRelativePath" just joins "input.cwd" with path to failed tests json